uportal-war/src/main/resources/properties/ehcache.xml

```diff
@@ -1922,7 +1922,14 @@
      +-->
     <cache name="org.jasig.portal.groups.pags.dao.EntityPersonAttributesGroupStore.entityGroup"
```

Comment:
On all of these caches that you add the cacheEventListenerFactory too, please remove from the comments 'Not replicated.' and replace with a separate line that says
'- replicated by invalidation'
like https://github.com/bjagg/uPortal/blob/1a0e8b78d17db9ee5268415cad5e29f7b7d05f1c/uportal-war/src/main/resources/properties/ehcache.xml#L1242

Also for those PAGS caches that don't have a comment, please add a comment like https://github.com/bjagg/uPortal/blob/1a0e8b78d17db9ee5268415cad5e29f7b7d05f1c/uportal-war/src/main/resources/properties/ehcache.xml#L408-L416